cmake/modules/AddCephTest.cmake

```diff
@@ -3,8 +3,8 @@
 #adds makes target/script into a test, test to check target, sets necessary environment variables
 function(add_ceph_test test_name test_path)
   add_test(NAME ${test_name} COMMAND ${test_path})
-  add_dependencies(check ${test_name})
-  set_property(TEST 
+  add_dependencies(tests ${test_name})
```

Comment:
@cbodley i missed the line at https://github.com/ceph/ceph/pull/10283/commits/ce5724effb270b0f57369fd1e9bd82d7cc872676#diff-9e70c23860593dc53da7c47ae633bbe7R481 . @alimaredia has added the dependencies to "make check".